OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/backend/impl/SubscriptionBackendService.kt

```diff
@@ -64,18 +64,20 @@ internal class SubscriptionBackendService(
             throw BackendException(response.statusCode, response.payload, response.retryAfterSeconds)
         }
 
-        val responseJSON = JSONObject(response.payload)
-        var rywToken: String? = null
-        if (responseJSON.has("ryw_token")) {
-            rywToken = responseJSON.getString("ryw_token")
-        }
+        fun JSONObject.safeString(key: String): String? = if (this.has(key)) this.getString(key) else null
 
-        var rywDelay: Long? = null
-        if (responseJSON.has("ryw_delay")) {
-            rywDelay = responseJSON.getLong("ryw_delay")
-        }
+        fun JSONObject.safeLong(key: String): Long? = if (this.has(key)) this.getLong(key) else null
```

Comment:
This is already defined in `JSONObjectExtension.kt`, you can just add `import com.onesignal.common.safeLong` to the top of this file to get this.